websocket_server.py

```diff
@@ -0,0 +1,59 @@
+import asyncio
+import websockets
+import serial

```

Comment:
We probably want to use ws or sockets.io for the web socket communication between the electroblocks.org website and the electron app.  The reason is that we will want to start on stop the python server every time we get new code from the website.  We want open communication between the app and site.